presidio_evaluator/dataset_formatters/i2b2_formatter.py

```diff
@@ -0,0 +1,55 @@
+from pathlib import Path
+from presidio_evaluator.data_objects import Span
+from typing import List, Optional, OrderedDict
+
+import collections
+
+import os
+
+import xmltodict
+
+from presidio_evaluator import InputSample
+from presidio_evaluator.dataset_formatters import DatasetFormatter
+
+class I2B22014Formatter(DatasetFormatter):
+  def __init__(
+        self,
+        files_path=Path("../../data/i2b2").resolve(),
+        glob_pattern: str = "*.*",

```

Comment:
If you're not using glob then this is probably not needed.

Reply:
We'll fix it later on

Reply:
forgot to remove it